src/arrayMethodJoin.js

```diff
@@ -4,9 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let NewString = '';
```

Comment:
don't name variables with uppercase letter

Reply:
agree, fix this comment